games-arcade/vvvvvv_data/vvvvvv_data-2.0.recipe

```diff
@@ -0,0 +1,35 @@
+SUMMARY="Data for VVVVVV"
+DESCRIPTION="Notice requested by the author: VVVVVV is a commercial game! The author has \
+given special permission to make this Haiku version available for free. If you \
+enjoy the game, please consider purchasing a copy at thelettervsixtim.es. \
+
+Data file from the Make and Play Edition of VVVVVV. Distributed with permission from \
+Terry Cavanagh as a license exception. More info available here: \
+https://github.com/TerryCavanagh/VVVVVV/blob/master/License%20exceptions.md"
+HOMEPAGE="https://thelettervsixtim.es"
+COPYRIGHT="Terry Cavanagh 2010-2020"
+LICENSE="VVVVVV Source Code License v1.0"
+REVISION="1"
+SOURCE_URI="https://thelettervsixtim.es/makeandplay/data.zip#noarchive"
+CHECKSUM_SHA256="6fae3cdec06062d05827d4181c438153f3ea3900437a44db73bcd29799fe57e0"
+
+ARCHITECTURES="any"
+DISABLE_SOURCE_PACKAGE="yes"
+
+PROVIDES="
+	vvvvvv_data = $portVersion
+	"
+REQUIRES="
+	vvvvvv
+	"
+
+BUILD()
```

Comment:
Please remove BUILD, REQUIRES

Reply:
Why should REQUIRES be removed? The package is useless without the main `vvvvvv` package (and vice versa).

Reply:
Because on secondary architecture nothing provides "vvvvvv"
It provides "vvvvvv$secondaryArchSuffix", thus leading to a non "any" package?